app/views/locations/_form.html.erb

```diff
@@ -190,6 +190,7 @@
     <% end %>
     
     <% if params[:reload] %>setTimeout(function() { location.reload(); }, 3000);<% end %>
+    notyNotification('relax', 3000, 'success', 'topCenter', 'Location Saved');
```

Comment:
Hi @gauravahlawat81 I am not sure why this PR is not showing conflicting changes on this file, there were new changes merged to this file a few weeks ago..

https://github.com/publiclab/plots2/blob/f87860ca517d79c487570f871929a452d8ee8bce/app/views/locations/_form.html.erb#L193-L200

Reply:
Yup @RuthNjeri , this PR is quite old and I haven't rebased the new code on this branch. I think I will make some more changes and push the updated code again. Thanks for reminding me! :confetti_ball: